business/workloads.go

```diff
@@ -556,6 +572,126 @@ func parseLogLine(line string, isProxy bool, engardeParser *parser.Parser) *LogE
 	}
 
 	// override the timestamp with a simpler format
+	timestamp := parseTimestamp(parsedTimestamp)
+	entry.Timestamp = timestamp
+	entry.TimestampUnix = parsedTimestamp.UnixMilli()
+
+	return &entry
+}
+
+func parseZtunnelLine(line string) *LogEntry {
```

Comment:
This is just a fundamental question... It looks like we are trying to treat these ztunnel logs to be handled like an Envoy access log.  Do we really need to do this parsing?  Or should be just treat them like "app" logs, just text lines?  Or maybe just try and parse out the timestamp?

Reply:
As an improvement, we could create a custom zTunnel logs modal with all the parsed information specific to the ztunnel

Reply:
I'm OK with the parsing as long as it's efficient enough, and reliable.  I was mainly just asking.